docs/semgrep-code/triage-remediation.md

```diff
@@ -132,10 +135,20 @@ To **reopen multiple findings** in the **No grouping** view, follow these steps:
 
 </details>
 
-## Ignore findings through GitHub PR comments
+## Ignore findings through PR and MR comments
+
+Triage your Semgrep AppSec Platform findings displayed as comments in GitHub PRs and GitLab MRs by replying with another comment.
 
-Triage your Semgrep AppSec Platform findings displayed as comments in GitHub PRs by replying with another comment.
+<Tabs
+    defaultValue="current"
+    values={[
+        {label: 'GitHub', value: 'current'},
+        {label: 'GitHub (beta)', value: 'gh'},
+        {label: 'GitLab (beta)', value: 'gl'}
+    ]}
+>
 
+<TabItem value='current'>
 :::info Prerequisites
```

Comment:
We should probably make this consistent with the other tabs (i.e. not a callout box)